config/default-amo.js

```diff
@@ -118,5 +118,5 @@ module.exports = {
   authTokenValidFor: 2592000, // 30 days
 
   // Turn on recommendations in details pages.
-  enableAddonRecommendations: true,
+  enableFeatureAddonRecommendations: true,
```

Comment:
Nit: I know it's not really what this patch is about, but I wonder if it would make sense to remove any flags that are no longer needed. I don't think we really need this `enableAddonRecommendations` flag anymore, as it was just in place to control the feature during the initial development/rollout. Rather than make a bunch of changes to a flag that actually should be removed, it might be more efficient to just remove it.

Reply:
That's a good idea. Usually, we have an issue for that. If the issue is missing for this flag, then I'll file one and work on it. I don't want to introduce too much changes in this PR.

Reply:
See: #6374